src/constants/interface-routes/sunbird-mentoring.json

```diff
@@ -0,0 +1,3335 @@
+{
```

Comment:
@rakeshSgr Let's move these files in project root, in sunbird-mentoring and mentoring